framework/resources/Functional/hive/hive_storage/hive_maprdb_json/plan/hive_maprdb_json_plan.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
We don't need 'data' and 'plan' folders, since the extensions are different anyway. 
For data, it will pick up .sql and .e_tsv files
For plan, it will pick up .sql.explain and .plan files
Can you please combine them into the same dir?

Reply:
Are you sure? I think it's better to have a single test suite per folder. They have a clearer look that way and easier to maintain.

Reply:
Yes, sure. That was the whole point of suggesting unique extensions. We are adding un-necessary levels of nesting directories otherwise. Also, it's easy to map a data query with a plan query, as they would now share the same name (with different extension). So we can have sets of 4 files per test which require both data and plan verification. It's a huge task to rework this later, so lets get this done when we are adding them.

Reply:
Done.